recipe/meta.yaml

```diff
@@ -27,6 +27,7 @@ requirements:
   host:
     - python
     - pip
+    - setuptools
     - cython
     - numpy
     - libgdal
```

Comment:
@akrherz, I'm not sure if there's gonna be a 1.4.0rc3 or 1.4.0 will be next, but could you make sure we switch from `libgdal` to `libgdal-core` when the next release happens? Xref https://github.com/conda-forge/rasterio-feedstock/issues/303

Reply:
Yeah, I reviewed that issue today and was trying to make a mental note to make that happen once 1.4.0 comes out!